pennylane/math/multi_dispatch.py

```diff
@@ -846,12 +846,46 @@ def norm(tensor, like=None, **kwargs):
             axis_val = kwargs.pop("axis")
             kwargs["dim"] = axis_val
 
+    elif like == "autograd":
+        ord = kwargs.get("ord", None)
+        if ord is not None and ord != "fro" and ord != "nuc":
+            norm = _autograd_norm

```

Comment:
Ah ok, so these are the norms that are not differentiable in autograd?

Reply:
Yep, basically. But the use case for other integer norms is limited, so I'm just going to dispatch to autograd for `ord=None` and scipy otherwise.